From 0ec3a1ce69b88e35d4cbd4410b14867b33322308 Mon Sep 17 00:00:00 2001
From: jakecoll <jakecoll@uchicago.edu>
Date: Tue, 19 Nov 2019 10:03:01 -0600
Subject: [PATCH 2/2] Add support for FN I/O Aggregator

---
 netmiko/ssh_dispatcher.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/netmiko/ssh_dispatcher.py b/netmiko/ssh_dispatcher.py
index 2b35fa3..1f45300 100644
--- a/netmiko/ssh_dispatcher.py
+++ b/netmiko/ssh_dispatcher.py
@@ -106,6 +106,7 @@ CLASS_MAPPER_BASE = {
     "cisco_xr": CiscoXrSSH,
     "cloudgenix_ion": CloudGenixIonSSH,
     "coriant": CoriantSSH,
+    "dell_fnioa": DellForce10SSH,
     "dell_dnos9": DellForce10SSH,
     "dell_force10": DellForce10SSH,
     "dell_os6": DellDNOS6SSH,
-- 
2.17.1

